tools/Modules/Run-UnitTests.ps1

```diff
@@ -34,10 +34,10 @@ function Test-Azure {
     )
     $defaults = [System.IO.Path]::GetDirectoryName($PSScriptRoot)
     Start-Job -ScriptBlock {
-        $ARMRoot = "$($using:PSScriptRoot)\..\..\src\Package\$BuildConfig\ResourceManager\AzureResourceManager"
-        Import-Module "$ARMRoot\AzureRM.Profile\AzureRM.Profile.psd1;"
-        Import-Module "$ARMRoot\AzureRM.Resources\AzureRM.Resources.psd1";
-        Set-Location $using:defaults;
+        $ARMRoot = "$($using:PSScriptRoot)\..\..\src\Package\$($using:BuildConfig)\ResourceManager\AzureResourceManager"
```

Comment:
Even with this change, it doesn't look like buildconfig resolved. 

`The specified module 'C:\workspace\powershell\tools\Modules\..\..\src\Package\\ResourceManager\AzureResourceManager\AzureRM.Profile\AzureRM.Profile.psd1;' was not loaded because no valid module file was found in any module directory.`

Reply:
It seems this CI job still use old HEAD rather than my commit.  @viananth Do you known why?

![image](https://user-images.githubusercontent.com/43633998/63249868-e8ec5580-c29c-11e9-8b49-a628aeddd5ea.png)
![image](https://user-images.githubusercontent.com/43633998/63249974-27821000-c29d-11e9-92ce-e68f9988c759.png)